Fix end of files with `end-of-file-fixer`

fabcor-maxiv opened the pull request:
Some files were set to be ignored by the `end-of-file-fixer` pre-commit hook.

This commit clears the list of files to be ignored by this hook, while still ignoring the `.hkli` and `.nml` files, and fixes the files accordingly. Of course, the fixes are done by the hook itself.

fabcor-maxiv commented:
Pushing this commit made GitHub warn me about the following:

> remote: warning: See https://gh.io/lfs for more information.
> remote: warning: File mxcubecore/configuration/mockup/gphl/test_samples/thermolysin/sample.hkli is 50.18 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
> remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.

I assume the file was already this big before.

fabcor-maxiv commented:
I have no clue if the change is meaningful for `*.hkli` and `*.nml` files. I do not know these file formats.

elmjag commented:
> Pushing this commit made GitHub warn me about the following:
> 
> > remote: warning: See https://gh.io/lfs for more information.
> > remote: warning: File mxcubecore/configuration/mockup/gphl/test_samples/thermolysin/sample.hkli is 50.18 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
> > remote: warning: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
> 
> I assume the file was already this big before.

Not sure if it's ok to change .hkli files. Maybe they are supposed to be like that, as per HKLI specification. @rhfogh I guess knows these details.

rhfogh commented:
Two points here:

1) .nml and .hkli files should be exempted from all linting. These are particular, messy formats (.nml is (gasp!) FORTRAN namelist format) and not messed with by anyone except GPhL. We use them for configuration.

2) The hkli files I use for mock testing - they support our acquisition emulation, generating images. They should not be modified, but maybe we should consider moving them out of the repository.

beteva commented:
Thanks. I reckon it will be great if the concerned sites remove old or irrelevant configuration files before merging this.

marcus-oscarsson commented:
Looks good, we should remove the `hkli` and `nml` files I guess and then we can merge, right @rhfogh ?

fabcor-maxiv committed:
Fix end of files with `end-of-file-fixer`

Some files were set to be ignored
by the `end-of-file-fixer` pre-commit hook.

This commit clears the list of files to be ignored by this hook,
while still ignoring the `.hkli` and `.nml` files,
and fixes the files accordingly.
Of course, the fixes are done by the hook itself.

fabcor-maxiv commented:
OK. The pull request is redone. The pre-commit hook now ignores only the `.hkli` and `.nml` files.
